src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Blocks/Table/headers_row.html.twig

```diff
@@ -0,0 +1,38 @@
+{#**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+<tr class="column-headers">
+  {% set orderBy, orderWay = grid.sorting.order_by, grid.sorting.order_way %}
+
+  {% for column in grid.columns %}
+    <th scope="col">
+      {% if column.options.sortable is defined and column.options.sortable %}
+        {{ ps.sortable_column_header(column.name, column.id, orderBy, orderWay) }}
```

Comment:
headers are static at the moment. could we allow creating templates for them too?

if we could use twig blocks for columns, maybe we could use blocks for headers as well?
Example: we have column with type `severity_level`. In template, we look for `column_severity_level_content` when rendering column content and for header we can look for `column_severity_level_header`, if it's not found, we can fallback to default template, wdyt @mickaelandrieu ?

Reply:
good idea, but not in this pull request